### ISSUE TITLE
Convert To Basic

### ISSUE THREAD

**Issue by nemchik** [msfeldstein/chrome-media-keys#57]
There are 20+ non basic controllers as of posting this issue. It might be a good idea to investigate which ones can be converted to basic to make long term maintenance easier. This might be better as a readme or wiki page that can be updated over time as controllers are converted to basic.
Here is a list of all non basic controllers;

AudioGalaxy
BronyTunes
Exfm
GetWorkDone
Hypemachine
Lastfm
Mog
Monstercat
Netflix
PonyFM
PonyvilleLive
Qriosity
Rbma
Shim (isn't this the Unity API?)
Shuffler
Soundtracker
Spotify (in progress)
Sway
Thesixtyone
ThisIsMyJam
Turntable
WeAreHunted
YouTube

**Comment by nemchik** [comment]
Removed Netflix https://github.com/msfeldstein/chrome-media-keys/pull/58 and Spotify https://github.com/msfeldstein/chrome-media-keys/pull/56

**Comment by mbirth** [comment]
Despite YouTube I have no account at any of those services so I couldn't check if the converted controller is working. Some look easy enough that the conversion should be possible "blind", but I wouldn't want to risk breaking things.

**Comment by Ustice** [comment]
There are a bunch of really old controllers. I removed the ones that are for services that are defunct in #104, but I am sure that there are a bunch that no longer work. I would be good to do an audit of the controllers to see which no longer work, and prioritize the services. 

Since Google Analytics is included already, I would suggest that we add it to the BasicController to see which services are _actually_ used.

**Comment by nemchik** [comment]
I completely agree about analytics, but is the base controller the right place to put it? Wouldn't that lack information about the controllers that are not basic?

**Comment by Ustice** [comment]
Is there a reason to not base all of the controllers on the BasicController while we are doing this?

**Comment by msfeldstein** [comment]
Yeah BasicController was added later, there are some legacy controllers
that don't use it, but there's no reason that they can't be converted to
use it.  If there's some reason a certain service can't use BasicController
file an issue and we can augment BasicController

**Comment by nemchik** [comment]
I updated the list to remove the controllers dropped with #104 

The only reason any of the remaining controllers are still not basic is either I didn't have access to the site to test a conversion, or I attempted a conversion and couldn't get it quite right.

I agree that blind conversions are possible, but risky. If anyone else wants to finish off the list to convert, feel free.
